packages/gatsby-plugin-utils/package.json

```diff
@@ -68,7 +69,8 @@
     "typescript": "^4.7.2"
   },
   "peerDependencies": {
-    "gatsby": "^4.0.0-next"
+    "gatsby": "^4.0.0-next",
+    "graphql": "^14.2.0 || ^15.0.0 || ^16.0.0"
```

Comment:
Why did you choose v14, v15 and v16? Gatsby currently only uses v15

Reply:
I got the range from `graphql-compose`, want me to change it to only include v15?

Reply:
I think this would make sense, yes. Multiple differing versions of `graphql` are problematic, so we wouldn't people to use v14 or v16 with Gatsby.

Thanks!